api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.3.x.xml

```diff
@@ -92,4 +92,22 @@
 			<where>system_id='admin'</where>
 		</update>
 	</changeSet>
+    
+    <changeSet id="TRUNK-5728-2020-04-15" author="samuel34">
+    	<preConditions onFail="MARK_RAN">
+    		<not>
+    			<columnExists tableName="conditions" columnName="encounter_id"/>
+    		</not>
+    	</preConditions>
+    	<comment>Adding 'encounter_id' column to 'conditions' table</comment>
+    	<addColumn tableName="conditions">
+    		<column name="encounter_id" type="int">
+    			<constraints nullable="true" />
+    		</column>
+    	</addColumn>
+    	<addForeignKeyConstraint constraintName="encounter_id_fk"
```

Comment:
The `encounter_id_fk` name is already taken.

Reply:
Sure